enhancements/must-gather-operator-disable-upload.md

```diff
@@ -0,0 +1,330 @@
+---
+title: must-gather-operator-disable-upload
+authors:
+  - "@rausingh"
+reviewers:
+  - "@TrilokGeer"
+  - "@Prashanth684"
+approvers:
+  - "@TrilokGeer"
+  - "@Prashanth684"
+api-approvers:
+  - "@TrilokGeer"
+  - "@Prashanth684"
+creation-date: 2025-09-03
+last-updated: 2025-09-03
+tracking-link:
+  - https://issues.redhat.com/browse/MG-86
+  - https://issues.redhat.com/browse/MG-67
+status: implementable
+see-also:
+---
+
+# Must Gather Operator Enhancement: DisableUpload in MustGatherSpec
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Add an optional field `disableUpload` to `MustGather.spec` that allows users to control whether the must-gather bundle should be uploaded to the SFTP server. When set to `true`, the bundle will be collected but not uploaded, providing users with the flexibility to handle the collected data locally or through alternative means.
+
+Supported values: `true`, `false`
+Default: `false` (upload enabled by default for backward compatibility)
+
+## Motivation
+
+### User Stories
+
+* As a cluster admin, I want to collect must-gather data without automatically uploading it to Red Hat's case management system so I can review the data locally first.
```

Comment:
The disconnected case could also be a relevant use case where you don't have the ability to upload to an outside env